### ISSUE TITLE
{CPAGERELATED}

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#3448]
`{CPAGERELATED: types=page,news&limit=3}`

If I use only news,  it works.
with page, news (both), limit is ignored

**Comment by Moc** [comment]
I assume the desired behaviour is to have the limit work for all types. 

For example when set to 3, it could show 1 news item and 2 pages. Rather than 3 news items and 3 pages. Correct?

**Comment by Jimmi08** [comment]
@Moc yes, it should work this way.  Or don't allow to select from both tables.
